<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tastierinoView</class>
 <widget class="QWidget" name="tastierinoView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn9">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnComma">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>,</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn0">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPoint">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
